app/dashboards/person_dashboard.rb

```diff
@@ -20,7 +20,7 @@ class PersonDashboard < BaseDashboard
     chat_handle: Field::String,
     photo: PhotoField.with_options(admin_only: true),
     job_title: Field::String,
-    specialties: Field::MultiSelectField.with_options(
```

Comment:
These changes seem unrelated to Finding Aid?

Reply:
Without this namespace, I was getting administrate errors due to association of finding aid with person model. Setting the namespace resolved the error.

Reply:
Person still works as expected.